scripts/auth_init.py

```diff
@@ -142,7 +142,7 @@ def create_client_app_payload(server_app_id: str, server_app_permission_setup_pa
             "redirectUris": ["http://localhost:50505/.auth/login/aad/callback"],
             "implicitGrantSettings": {"enableIdTokenIssuance": True},
         },
-        "spa": {"redirectUris": ["http://localhost:50505/redirect"]},
+        "spa": {"redirectUris": ["http://localhost:50505/redirect", "http://localhost:5173/redirect"]},
```

Comment:
is 5173 a port used by codespaces, or was it a local choice?

Reply:
Ah that's what vite dev uses! I often use "npm run dev" when working on the frontend in order to get hot reloading.